packages/client/hmi-client/src/workflow/tera-workflow.vue

```diff
@@ -67,7 +67,11 @@
 						@append-output-port="(event: any) => appendOutputPort(node, event)"
 						@append-input-port="(event: any) => appendInputPort(node, event)"
 						@update-state="(event: any) => updateWorkflowNodeState(node, event)"
-					/>
+					>
+						<template #placeholder-graphic>
+							<tera-operator-placeholder-graphic :operation-type="node.operationType" />
```

Comment:
This should not be there, the developer has to add it to the `node` itself.
If a new node is created without one, that's a bug, we should not prepare for that.